ObjCVoiceQuickstart/ViewController.m

```diff
@@ -257,9 +275,9 @@ - (void)callDidConnect:(TVOCall *)call {
     
     [self.placeCallButton setTitle:@"Hang Up" forState:UIControlStateNormal];
     
-    [self toggleUIState:YES];
+    [self toggleUIState:YES showCallControl:YES];
     [self stopSpin];
-    [self routeAudioToSpeaker];
+    [self toggleAudioRoute:YES];
```

Comment:
One question I have about the flow of this app - can we apply the appropriate audio route when `configureAudioSession` is called instead?